docs/architecture/adr-104-out-of-band-state-sync.md

```diff
@@ -271,68 +330,125 @@ snapshot_dump_path=""
 
 CometBFT exposes a CLI command to instruct the application to dump the existing snapshots into an exportable format:
 
 ```bash
 dump      Dump existing snapshots to exportable file format.
 ```
 
-We can expand the CLI interface to allow for additional operations, similar to the above provided CLI. However,
-as snapshot generation and dumping would be built in into CometBFT, while the node is running, we do not need
-to rely on a CLI for this functionality.
-
-Applications can themselves be in charge of dumping the snapshots into a given file format. An alternative solution is 
-that CometBFT, itself, after recieveing snapshots obtained via `ListSnapshots`, generates a file and dumps it to 
-a predefined location. 
-
-On startup, if `local_sync` is set to `true`, CometBFT will try to look for an existing snapshot at the path
-given by the operator. The snapshots will be loaded and state restored as if they came from a peer in the current implementation. 
+We could expand the CLI interface to allow for additional operations, similar to the above provided CLI. However,
+as snapshot generation, dumping and loading a local snapshot would be built into CometBFT,
+ while the node is running, we do not need to rely on a CLI for this functionality.
 
 The `dump` command can be implemented in two ways:
-- Rely on the existing ABCI functions `ListSnapshots` and `LoadChunks` to retrieve the snapshots and chunks from a peer. 
-This approach requires no change to the current API and is easy to implement. It however involves more than one ABCI call and 
-data transafer. 
-- Extend `RequestListSnapshots` with a flag to indicate that we want an exportable snapshot format and extend `ResponseListSnapshot` to return a 
+
+1. Rely on the existing ABCI functions `ListSnapshots` and `LoadChunks` to retrieve the snapshots and chunks from a peer. 
+This approach requires no change to the current API and is easy to implement. Furthermore, CometBFT has complete control
+over the format of the exported snapshot. It does however involve more than one ABCI call and network data transfer. 
+2.  Extend `RequestListSnapshots` with a flag to indicate that we want an exportable snapshot format and extend `ResponseListSnapshot` to return a 
 path and format of the exported snapshots.
-- Add path parameters to the command, and include the path into `RequestListSnapshots` instructing the application to generate a snapshot export
-at a given location. 
+
+An improvement to the second option mentioned above would be to add path parameters to the command,
+ and include the path into `RequestListSnapshots` instructing the application to generate a snapshot export
+at a given location.
+
+A third option is the introduction of a new ABCI call: `ExportSnapshots`, which will have the same effect as option 2 above.
+
+
+#### *Automatic snapshot exporting*
+
+Applications generate snapshots in fixed time intervals. The application itself measures the time passed since the last snapshot,
```

Comment:
```suggestion
Applications generate snapshots with a regular cadence (fixed time intervals or heights). The application itself measures the time or number of heights passed since the last snapshot,
```